job_templates/install_group_-_katello_ansible_default.erb

```diff
@@ -0,0 +1,16 @@
+<%#
+name: Install Group - Katello Ansible Default
+snippet: false
+model: JobTemplate
+job_category: Katello
+description_format: Install package group(s) %{package}
+template_inputs:
+- name: package
+  required: true
+  input_type: user
+  advanced: false
+provider_type: Ansible
+kind: job_template
+%>
+
+<%= render_template('Run Command - Ansible Default', :command => "yum group install #{input('package')}") %>
```

Comment:
many files does not have EOF character, it should be added I think

Reply:
Sorry, I'm not sure what do you mean here? What do EOF characters have to do with `yum group install`?

Reply:
nothing :-) just that EOL was missing and this was last line in the file